### PR TITLE
Allow to optionally delete data from managed folders and/or job and scenario logs …

### PR DESCRIPTION
**Pull request by vojtech2**
…when deleting a project

[sc-71959](https://app.shortcut.com/dataiku/story/71959/ability-to-delete-job-logs-when-deleting-project)

Also see: dataiku/dip#13848

### CODE REVIEW

**Review by pbailly — APPROVED** [review]
LGTM, but don't merge before 10.0.0 release.

File: dataikuapi/dss/project.py [hunk 1]
```diff
@@ -79,17 +79,21 @@ def move_to_folder(self, folder):
     # Project deletion
     ########################################################
 
-    def delete(self, drop_data=False):
+    def delete(self, drop_data=False, drop_managed_folders_output_of_recipe=False, delete_job_logs=False):
```
Comment (pbailly) [inline dataikuapi/dss/project.py:82]:
```suggestion
    def delete(self, drop_data=False, drop_managed_folders_output_of_recipe=False, delete_job_logs=True):
```

File: dataikuapi/dss/project.py [hunk 2]
```diff
@@ -79,17 +79,25 @@ def move_to_folder(self, folder):
     # Project deletion
     ########################################################
 
-    def delete(self, drop_data=False):
+    def delete(self, clear_managed_datasets=False, clear_output_managed_folders=False, clear_job_and_scenario_logs=True, **kwargs):
```
Comment (vojtech2) [inline dataikuapi/dss/project.py:82]: I found many instances in integration tests (and beyond) where we call this with `drop_data=True` or just with one positional argument `True`. Should I go ahead and change these calls?

Comment (vojtech2) [reply dataikuapi/dss/project.py:82]: Done.

**Review by pbailly — APPROVED** [review]
LGTM

**Pull request merged**